sys/include/net/ble.h

```diff
@@ -97,6 +97,8 @@ extern "C" {
 #define BLE_GATT_SVC_HRS                    (0x180d)    /**< heart rate service */
 #define BLE_GATT_SVC_BAS                    (0x180f)    /**< battery service */
 #define BLE_GATT_SVC_IPSS                   (0x1820)    /**< IP protocol support */
+/* XXX: custom services not defined by the BT SIG (random values) */
```

Comment:
Is there a policy on how these should be added?

Reply:
no. Just add something if you think its sensible, easy as that.